res/skins/QMLDemo/Mixxx/Controls/Knob.qml

```diff
@@ -50,8 +51,8 @@ Item {
             fillColor: "transparent"
 
             PathAngleArc {
-                startAngle: -90
-                sweepAngle: (root.value - (root.max - root.min) / 2) * 2 * root.angle
+                startAngle: -90 + root.angle * 2 * (root.arcStart - (root.max - root.min) / 2)
+                sweepAngle: (root.value - root.arcStart) * 2 * root.angle
```

Comment:
This is very unreadable to me. I'd prefer to split these binding over multiple lines with named variables so its easier to understand what part of the equation does what. Or maybe I'm just too tired to understand right now...

Reply:
ab3c3e0f03b527f1a9520b979151dd9dbbff5bdd